app/src/main/java/hr/foi/air/servicesync/backend/FirestoreService.kt

```diff
@@ -0,0 +1,60 @@
+package hr.foi.air.servicesync.backend
+
+import com.google.firebase.firestore.FirebaseFirestore
+
+class FirestoreService {
+    private val db = FirebaseFirestore.getInstance()
+
+    fun saveReservation(
+        companyId: String,
+        serviceName: String,
+        reservationDate: Long,
+        userId: String,
+        onSuccess: () -> Unit,
+        onFailure: (Exception) -> Unit
+    ) {
+        val reservation = hashMapOf(
+            "companyId" to companyId,
+            "serviceName" to serviceName,
+            "reservationDate" to reservationDate,
+            "userId" to userId
+        )
```

Comment:
You could make a data class called Reservation.
`Class Reservation(companyId: String,
serviceName: String,
reservationDate: Date/Long,
userId : String`)

Reply:
Classic

Reply:
Noted for the next sprint.